content/blog/2024-07-31-valkey-8-0-0-rc1.md

```diff
@@ -0,0 +1,157 @@
++++
+# `title` is how your post will be listed and what will appear at the top of the post
+title= "Valkey 8.0: Delivering Enhanced Performance and Reliability"
+# `date` is when your post will be published.
+# For the most part, you can leave this as the day you _started_ the post.
+# The maintainers will update this value before publishing
+# The time is generally irrelvant in how Valkey published, so '01:01:01' is a good placeholder
+date= 2024-08-02 01:01:01
+# 'description' is what is shown as a snippet/summary in various contexts.
+# You can make this the first few lines of the post or (better) a hook for readers.
+# Aim for 2 short sentences.
+description= "Delivering significant improvements in throughput, scalability, and observability for modern data-intensive applications"
```

Comment:
is this the new title? seems quite lengthy

Reply:
No, I honestly don't know what this is. "Valkey 8.0: Delivering Enhanced Performance and Reliability"